js/icons.js

```diff
@@ -1,33 +1,39 @@
 //which icons to use
 let window_storage = window.localStorage;
+console.log(window_storage);
 icon_state_stored = window_storage.getItem("newicons");
 newIcons = true;
 if (icon_state_stored === "false") {toggleIcons()} 
 
+
 /** Toggle icons on the ENTIRE page.
  * 
  */
-function toggleIcons() {
+ function toggleIcons() {
     newIcons = !newIcons;
-    let imgs = document.getElementsByTagName("IMG");
+    let imgs = document.getElementsByTagName("img");
+    let divs = document.getElementsByTagName("div");
     let favicon = document.querySelector("link[rel~='icon']");
-    let toggleiconbutton = document.getElementById("toggle-icon-button");
 
     if (newIcons) { //switch to new
         favicon.href = favicon.href.replace("media/icons/old","media/icons/new");
         for (const img of imgs) {
             if (img.src.includes("media/icons/old")) {img.src = img.src.replace("media/icons/old","media/icons/new");}
-            if (img.src.includes("media/items/old")) {img.src = img.src.replace("media/items/old","media/items/new");}
         }
-        toggleiconbutton.textContent = "Use Old Icons";
+        for (const div of divs) {
+            if (div.style.backgroundImage.includes("media/items/old")) {div.style.backgroundImage = div.style.backgroundImage.replace("media/items/old","media/items/new");}
+        }
+        //toggleiconbutton.textContent = "Use Old Icons";
```

Comment:
maybe label all the atlas images with a class or something? to make this a bit cleaner
https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName

though idk how this performs, if it does a linear scan internally or if there's something more optimized / internal data structures to speed it up

though it should be more optimized in any case (linear scan in native vs js)